HowToMigrateSchema.md

```diff
@@ -0,0 +1,61 @@
+1. Just try to run the "run.sh" or "run.bat" file. If it does migrate, good for you! You are done and don't need to do anyting <br/>
+
+If it is not working, then proceed to step2. It will be somewhat long process, but once you have done this, then you will not need to reset your schema any longer, so you don't need to do this again in the future!
+
+2. There could be a case (this is not an error) that your console has the following statements and does not update anything:
```

Comment:
Awesome, thanks for finding and documenting this

Reply:
No problem :)